lib/src/frame.dart

```diff
@@ -17,12 +17,36 @@ final _vmFrame = RegExp(r'^#\d+\s+(\S.*) \((.+?)((?::\d+){0,2})\)$');
 //     at VW.call$0 (eval as fn
 //         (https://example.com/stuff.dart.js:560:28), efn:3:28)
 //     at https://example.com/stuff.dart.js:560:28
-final _v8Frame =
+final _v8JsFrame =
```

Comment:
(While here, it may be worth saying that these regexps are applied to individual lines. That's why the `^` doesn't require `multiLine: true` and why it's not a problem that `[^\]+` can match newlines. Helps the reader set expectations.)

Reply:
I think it's clear from the use sites a few lines below that these parse a single "frame", as passed to factories like `Frame.parseVM(String frame)`.

However the exact format of those "frame"s is not described by this library even though the `Frame` type and its factories are public.

It would be good to describe the format for the users who may want to parse a single frame.

Let's do it separately though.